### PR TITLE
distinguish unit tests and integration tests

### PR DESCRIPTION
**Pull request by xuanyu66**
## What problem does this PR solve?
- Use Junit Category to distinguish unit tests and integration tests so that we can run tests separately.

## Related changes
- modify readme about run tests and how to write tests
- modify ci to run unit tests and integration tests separately.

### CODE REVIEW

**Comment by xuanyu66** [comment]
@sunxiaoguang @humengyu2012 PTAL

**Comment by xuanyu66** [comment]
@sunxiaoguang @humengyu2012 @shiyuhang0 @zhangyangyu PTAL

**Review by zhangyangyu — APPROVED** [review]

File: README_zh_CN.md [hunk 1]
```diff
@@ -39,7 +39,40 @@ export TIDB_HOST="127.0.0.1"
...
+```xml
+ <dependency>
+      <groupId>io.tidb</groupId>
+      <artifactId>test</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+</dependency>
+```
+
+然后，我们使用 Junit 的 Category 来区分集成测试，用法如下:
```
Comment (sunxiaoguang) [inline README_zh_CN.md:68]: Junit -> JUnit

**Review by sunxiaoguang — COMMENTED** [review]
Rest LGTM

**Review by sunxiaoguang — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -32,14 +32,43 @@ TiBigData project is under the Apache 2.0 license. See the [LICENSE](./LICENSE)
...
+```xml
+ <dependency>
+      <groupId>io.tidb</groupId>
+      <artifactId>test</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+</dependency>
+```
+
+Second, use Junit Category to clarify integration-test:
```
Comment (xuanyu66) [inline README.md:64]:
```suggestion
Second, use JUnit Category to clarify integration-test:
```

**Review by humengyu2012 — APPROVED** [review]
LGTM

**Comment by xuanyu66** [comment]
@sunxiaoguang @humengyu2012 PLZ merge this PR

**Pull request merged**